.github/workflows/main.yml

```
@@ -25,7 +25,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn
+      - run: yarn prebuild
       - run: yarn build
+      - run: yarn postbuild
```

You don't need to add `prebuild` and `postbuild`, they are called automatically upon running `build`.
```suggestion
      - run: yarn build
```